web/generate-geometries.js

```diff
@@ -419,6 +420,7 @@ const zoneDefinitions = [
   { zoneName: 'NF', type: 'country', id: 'NFK'},
   { zoneName: 'NG', type: 'country', id: 'NGA'},
   { zoneName: 'NI', type: 'country', id: 'NIC'},
+  { zoneName: 'NKR', type: 'country', id: 'NKR'},
```

Comment:
It needs to be in this form.

`{ zoneName: 'NKR', type: 'administrations', administrations: [insert adm1_codes here]},`

Reply:
The same will need to be done for `AZ`.

Reply:
In such case, the following rayons would go in NKR:
Kalbajar, Lachin, Qubadli, Zangilan, Jabrayil, Susha District, Susha City, Tartar, Khankendi, Khojali, Khojavend, Agdam. The only one strange is the Fizuli district that should be cut in half, but for simplicity we can leave it in main AZ. I'll try doing a commit for it.